packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -23,7 +21,6 @@ class CreateCommand extends Command {
   CreateCommand() {
     argParser.addOption('out',
       abbr: 'o',
-      hide: true,
```

Comment:
This option is hidden because it's going away - create is now `flutter create <out-dir>`.